Tools-Override/tests.targets

```diff
@@ -47,7 +47,8 @@
   <PropertyGroup>
     <XunitResultsFileName>testResults.xml</XunitResultsFileName>
 
-    <XunitOptions Condition="'$(BuildingNETFxVertical)' == 'true'">$(XunitOptions) -noshadow -noappdomain </XunitOptions>
+    <XunitOptions Condition="'$(BuildingNETFxVertical)' == 'true'">$(XunitOptions) -noshadow</XunitOptions>
```

Comment:
> We were always passing noappdomain before but now it is only happening in the one test project. Is this going to cause other test projects to fail?

we used to not pass -noappdomain before. we just added after the engineering work and we still going through the tests to fix what is failing. but in general I am not expecting this will cause a problem and if does we can still go back and reverse setting it.